packages/react-vapor/src/components/slider/SliderHandle.tsx

```diff
@@ -0,0 +1,34 @@
+import {Handle} from 'rc-slider';
+import Tooltip from 'rc-tooltip';
+import * as React from 'react';
+import * as _ from 'underscore';
+
+export interface CustomHandleProps {
+    customTooltip: React.ReactNode;
+    hasTooltip: boolean;
+    rangeOutput: number;
+}
+
+export interface HandleProps {
+    dragging: boolean;
+    vertical: boolean;
+    offset: number;
+    reverse: boolean;
+    value: number;
+}
+
+const SliderHandle: React.FunctionComponent<{handleProps: HandleProps; handleCustomProps: CustomHandleProps}> = ({
+    handleProps,
+    handleCustomProps,
+}) => (
+    <Tooltip
+        prefixCls="rc-slider-tooltip"
+        overlay={handleCustomProps.customTooltip ? handleCustomProps.customTooltip : handleCustomProps.rangeOutput}
```

Comment:
```suggestion
        overlay={handleCustomProps.customTooltip ?? handleCustomProps.rangeOutput}
```

Reply:
:soNullishCoalescingAdvocate: that is powerful stuff. :heart: